File: コーディングによる問題解決/03 アルゴリズム 初級/全探索/04_正の整数の個数/4.js
const fs = require("fs");
const input = fs.readFileSync(0, "utf8").trim().split(/\s+/);

const N = Number(input[0]);
const A = input.slice(1).map(Number);

let flag = false;
let cnt = 0;
for (let i = 0; i < N; i++){
    if (A[i] > 0){
        cnt++;
    }
}

console.log(cnt);

File: Coding/03_Beginner/Array/Python/4.py
N = int(input())
A = list(map(int,input().split()))
Q = int(input())
for i in range(Q):
    query = list(map(int,input().split()))
    if query[0] == 0:
        k = query[1]
        if len(A) <= k:
            print("Error")
        else:
            print(A[k])
    else:
        v = query[1]
        A.append(v)